disorder-core/src/Disorder/QuickCheck.hs

```diff
@@ -0,0 +1,6 @@
+module Disorder.QuickCheck (
```

Comment:
yeah I am honestly not sure where or if this should live, was more to make a point!